### PR TITLE
updating test in TestMultipleSchedulers test suite to avoid race condition

### PR DESCRIPTION
**Pull request by sakshamgarg**
#### Bug/feature Description
* *TestMultipleSchedulers.test_set_msched_update_inbuilt_attrs_accrue_type fails due to race condition*

#### Affected Platform(s)
* *All*

#### Cause / Analysis / Design
* *The test fails sometimes as the second submitted job gets started before it is being able to check for accrue_type=2.*

#### Solution Description
* *Running both the submitted jobs for default sleep time.*
* *Deleting first job after checking the accrue_type of second submitted job.*

#### Testing logs/output
* *[ptl_output.txt](https://github.com/PBSPro/pbspro/files/2794908/ptl_output.txt)*

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
   - [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
   - [ ] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
- [x] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [x] I have attached **test logs to this pull request** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

File: test/tests/functional/pbs_multi_sched.py [hunk 1]
```diff
@@ -1904,6 +1902,7 @@ def test_set_msched_update_inbuilt_attrs_accrue_type(self):
         # accrue_type = 2 is eligible_time
         self.server.expect(JOB, {ATTR_accrue_type: 2}, id=jid2)
 
+        self.server.delete(jid1)
```
Comment (anamikau) [inline test/tests/functional/pbs_multi_sched.py]: add wait=true to wait for job1 to get dequeued.

**Comment by visheshh** [comment]
Looks good to me

**Comment by sakshamgarg** [comment]
Thanks for the review. Should I rebase and squash my commits ?

**Comment by anamikau** [comment]
@sakshamgarg yes please squash and rebase.

**Comment by sakshamgarg** [comment]
@anamikau I have rebased and squashed.

**Comment by anamikau** [comment]
Thanks @sakshamgarg. merged the changes in

**Pull request merged**